apps/backend/src/lib/emails.tsx

```diff
@@ -46,10 +46,9 @@ export async function getEmailTemplateWithDefault(projectId: string, type: keyof
   };
 }
 
-function getPortConfig(port: number | string) {
+export function isSecureEmailPort(port: number | string) {
   let parsedPort = parseInt(port.toString());
-  const secure = parsedPort === 465;
-  return { secure };
+  return parsedPort === 465;
 }
```

Comment:
you can't determine whether an SMTP server uses TLS or not just by the port (eg. it may be on a different port such as 587, 2525, 25025, etc). Also, 465 is not recommended anyways: https://www.mailgun.com/blog/email/which-smtp-port-understanding-ports-25-465-587/

Reply:
Should we add a setting in the DB for TLS?